rbac/management/role/view.py

```diff
@@ -449,8 +449,8 @@ def validate_role(self, request):
                     error = {key: [_(message)]}
                     raise serializers.ValidationError(error)
 
-                db_permission = Permission.objects.filter(
-                    application=app, resource_type=resource_type, verb=verb
+                db_permission = filter_queryset_by_tenant(
+                    Permission.objects.filter(application=app, resource_type=resource_type, verb=verb), request.tenant
```

Comment:
Just documenting from our chat in slack, that we need to keep any `Permission` queries the way they are, not filtered by `tenant`. This is because we currently have one single set of permission objects in the public schema, belonging to the public tenant. We should just return everything, as permissions are not allowed to be created per tenant, via the api or otherwise. If we filter by `request.tenant` here, we'll get an empty queryset once we flip to serve from public.